### PR TITLE
Added GET version of /search/stac

### PR DESCRIPTION
**Pull request by hgs-msmith**
This a proposed simple alternative to the POST method.

### CODE REVIEW

**Comment by hgs-msmith** [comment]
BTW, if we like this, I'll add it to WFS3core+STAC.yaml also.

File: api-spec/STAC-standalone.yaml [hunk 1]
```diff
@@ -19,8 +19,39 @@ servers:
     description: Production server
 paths:
   /search/stac:
+    get:
+      summary: >-
+        retrieve items matching filters. Intended as a shorthand API for simple queries.
```
Comment (cholmes) [inline api-spec/STAC-standalone.yaml]: I'd throw a little '(optional)' bit in here so implementors know it's not required.

Comment (hgs-msmith) [reply api-spec/STAC-standalone.yaml]: Looks good - I'll add this to WFS3core+STAC.yaml also.

**Comment by hgs-msmith** [comment]
Fixes #86 
Fixes #90

**Pull request merged**